app/javascripts/gencat/modules/gp_gencat_map_controller.js

```diff
@@ -25,157 +37,309 @@ window.GobiertoPeople.GencatMapController = (function() {
   return GencatMapController;
 })();
 
+
 function createMap(options) {
+
   if ($('#map').length === 0) return;
+  const { fromDate } = options
+
+  mapboxgl.accessToken = 'pk.eyJ1IjoiYmltdXgiLCJhIjoiY2swbmozcndlMDBjeDNuczNscTZzaXEwYyJ9.oMM71W-skMU6IN0XUZJzGQ';
+
+  let dots
+  let featureElement
+  let initZoom
+  const buttonCloseTooltip = document.getElementById('tooltip--close')
+  buttonCloseTooltip.addEventListener('click', closeTooltip)
+  const dataGenCatTrips = 'https://gencat.gobify.net/api/v1/data/data.csv?sql=select%20*%20from%20trips'
```

Comment:
Make this URL relative, to don't depend on the domain (in production is a different one)